src/command-registry.coffee

```diff
@@ -239,6 +250,8 @@ class CommandRegistry
       break if propagationStopped
       currentTarget = currentTarget.parentNode ? window
 
+    @emitter.emit 'did-dispatch', syntheticEvent
```

Comment:
Yeah, I think `target` would be the field you'd want to check; not `currentTarget`.